### PR TITLE
Add steps for deploying an existing node.js app to OpenShift. Fixes R…

### PR DESCRIPTION
**Pull request by rhoads-zach**
…HOARDOC-1190

### CODE REVIEW

**Comment by rhoads-zach** [comment]
@lholmquist @Ladicek let me know what you think.

**Comment by lholmquist** [comment]
👍

File: docs/topics/proc_deploy-existing-nodejs-app.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+= Deploying an Existing {NodeJS} Application to OpenShift
+
+You can deploy an existing {NodeJS} application to OpenShift using the steps in xref:deploy-existing-nodejs-app-openshift[].
+
+In addition, you must verify the following items when deploying an existing application:
+
+* Ensure the `files` section of `package.json` lists of all files and directories you need to deploy with your application.
```
Comment (Ladicek) [inline docs/topics/proc_deploy-existing-nodejs-app.adoc]: "lists of all files" -> "lists all files"

File: docs/topics/proc_creating-nodejs-app.adoc [hunk 2]
```diff
@@ -65,6 +65,7 @@ $ curl http://localhost:8080
 {"content":"Hello, World!"}
 ----
 
+[#deploy-existing-nodejs-app-openshift]
```
Comment (Ladicek) [inline docs/topics/proc_creating-nodejs-app.adoc]: I'm not sure it's the best idea to call this section `deploy-existing-nodejs-app-openshift`, when you link to it from some other section that's also called "proc_deploy-existing-nodejs-app.adoc".

**Comment by Ladicek** [comment]
Otherwise LGTM.

**Comment by rhoads-zach** [comment]
thanks for the feedback @Ladicek 

fixed.

**Pull request merged**